matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/content/ContentAttachmentData.kt

```diff
@@ -36,7 +36,7 @@ data class ContentAttachmentData(
         val queryUri: Uri,
         val mimeType: String?,
         val type: Type,
-        val waveform: List<Int>? = null
+        val waveform: List<Int?>? = null
```

Comment:
This is only used locally, so I do not understand why you are changing the type here too.

Reply:
You are right. Done.